health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationScreenParent/DrawerFieldComposer.js

```
@@ -0,0 +1,459 @@
+
+    // Extract payload (supports multi-line JSON)
+    let payload = {};
+    const dataMatch = curl.match(/--data(?:-raw)?\s+(['"])([\s\S]+?)\1/);
+    if (dataMatch) {
+      const rawData = dataMatch[2]; // Extract JSON body
+      try {
+        payload = JSON.parse(rawData);
+
+        // Remove RequestInfo if present
+        if (payload.RequestInfo) {
+          delete payload.RequestInfo;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)